### DISCUSSION TITLE
Question: direct link between time step size and platoon size

### DISCUSSION THREAD

**Discussion by EwoutH** [toruseo/UXsim#148] [Q&A] [answered]
I noticed that the time step size (`DELTAT`) is directly linked to the platoon size (`DELTAN`) through the reaction time (`REACTION_TIME`), as shown in the initialization `DELTAT = REACTION_TIME * DELTAN`.

I'm curious why this is modelled this way, could you clarify the rationale behind this direct linkage between time step size and platoon size?

**Comment by toruseo** [comment, marked as answer]
There is a theoretical reason. In this way, we can compute the exact solution of KW model (the underlying traffic flow model of UXsim) efficiently. In short, it eliminates numerical errors. If we use different setting, it will induce so-called numerical diffusion or instability.

If you want know the details, you need to read a paper: http://www.sciencedirect.com/science/article/pii/S0191261513000349

**Comment by EwoutH** [comment]
Thanks!

By the way, it might be useful if the [Discussions](https://github.com/features/discussions) tab could be enabled on this repository. Then users would have a place to discuss questions, configurations and new ideas, without adding noise here in the issues.

It can be enabled very easily, see [Enabling GitHub Discussions on your repository](https://docs.github.com/en/discussions/quickstart#enabling-github-discussions-on-your-repository).
